reference/cli/create.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Create"
```

Comment:
Should we have the title be "Create Kernel App". To me, Create is not descriptive enough in the docs title?